integration/setup/provider.go

```diff
@@ -28,6 +31,8 @@ import (
 
 // provider installs the operator and tenant cluster CR.
 func provider(config Config) error {
+	ctx := context.Background()
```

Comment:
I'd say `provider` should take `ctx` instead.

Reply:
Yes but this is so unimportant right now and really not my problem. We will do this at some point but this shit has to work first. Right now it still does not.